.eslintrc.js

```diff
@@ -48,6 +48,7 @@ module.exports = {
       rules: {
         '@typescript-eslint/explicit-module-boundary-types': 0,
         '@typescript-eslint/no-non-null-assertion': 0,
+        '@typescript-eslint/no-explicit-any': 0,
```

Comment:
Hey @justinkambic I was wondering how do you feel about it?

Reply:
I'm not crazy about it because I feel like our usage of `any` will be unrestricted and therefore more likely. But if we agree to not utilize `any` in situations where it's unnecessary but convenient, I suppose it's alright.

I usually prefer to disallow `any` and cover it with `@ts-expect-error We're using any because....`.